plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -238,6 +238,44 @@ public void testTableNameClash()
         }
```

Comment:
What about testing views?

Reply:
We can address it in separate PR. Out of scope for this PR.